README.example.md

```diff
@@ -58,8 +36,19 @@ This section should include any business related explanation that helps understa
 
 Name, and a short description of any services that the project is using (error tracking, activity monitoring, log registry, email service, etc). Include where to get the credentials. E.g., 1Password under admin+project@abtion.com.
 
+# Requirements
+
+You must have the following installed and available on your machine:
+
+- **Ruby 2.6.5**
+- **Node JS 12.x**
```

Comment:
Add yarn?

Reply:
Good point, it makes sense to add yarn to the dependencies now that we are removing the instructions about how to install it :+1: